mockups/pain-a.html

```
@@ -0,0 +1,470 @@
+        <span class="time">14:30</span>
+      </div>
+      <div class="entry-meta">
+        <div class="tags-line">
+          <span class="tag-mini">head</span>
+          <span class="tag-mini">neck</span>
+          <span class="tag-mini">nausea</span>
+        </div>
+        <p class="note">Started after meeting</p>
+        <div class="fatigue-coffee">Fatigue 5 · Coffee 2</div>
+      </div>
+      <button class="entry-action">⋯</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)